docs/architecture/index.rst

```diff
@@ -0,0 +1,82 @@
+
+Single datasource: OIP obtains data from registration-components like Open Zaak and Open Klant
+
+Data ownership: Citizens have direct access to their data and can delete their profile at any time
+
+Standards & Modern IT: We make use of modern REST APIs and standards like the VNG ZGW APIs in order to allow easy integration and data portability.
+
+Community & Agile: The development of OIP is driven by 6 municipalities together with Dimpact in a per-sprint basis with monthly releases
+
+Architecture
+==================
+
+OIP is a Python/Django webapplication which provides a large degree of flexibility for the municipality using it. Customization is possible thanks to an extensive range of configurationsettings in the admin, combined with Django CMS for structuring the pages and page structure of the install.
+
+The Django code-base is structured into a range of built-in Django apps, one for each integration (`haalcentraal`, `openzaak`, `openklant` etc).
+
+For making use of these integrations in a pluggable fashion, OIP uses Django CMS apphooks which can be found in the `cms` app:
+
+ * `products` publishes the PDC / Category and Product details to anonymous and logged-in users (eg. Productenpagina's)
+ * `cases` exposes the Open Zaak case details to logged-in users (eg. Mijn aanvragen)
+ * `collaborate` exposes the cooperative planning module to logged-in users (eg. Samenwerken)
+ * `inbox` exposes the internal messaging app to logged-in users (eg. Mijn Berichten)
+ * `profile` exposes the profile to logged-in users (eg. Mijn Profiel)
+ * `benefits` exposes the ability to download benefit specifications via SSD to logged-in users(eg. Mijn Uitkeringen)
+
+Which of these `cms` apps are used is configured by activating a Django CMS apphook for that app via the CMS admin interface. Enabling the apphook enables the feature on the CMS page on which it is activated. Generic OIP Django apps like `accounts` are always active and provide overall authentication/authorization and user management features.
+
+ * For the frontend OIP makes use of SCSS and Javascript, with HTMX/django-htmx integration for specific usecases for dynamic content.
+ * For the database OIP makes use of PostgreSQL.
+ * For caching and task queues OIP makes use of Redis.
+ * For background tasks OIP uses Celery combined with Celery Beat.
+ * For search OIP integrates with ElasticSearch.
+
+With OIP we make use of a number of shared Maykin-built Django apps like django-digid-eherkenning, zgw-consumers, notifications-api-common, maykin-2fa, django-admin-index, django-open-forms-client, django-log-outgoing-requests, django-setup-configuration and mozilla-django-oidc-db.
+
+
+Datamodels
+==================
+
+Generated based on OIP v1.17
+
+Accounts app
+.. graphviz:: accounts.dot
+
+Configuration app	      
+.. graphviz:: configurations.dot
+
+Haal Centraal app
+.. graphviz:: haalcentraal.dot
+
+KvK app
```

Comment:
Perhaps not vital but "KVK" is now officially spelled with all-caps, instead of the old lower case little 'v'.

Reply:
Thanks!